### PR TITLE
updated job launch mechanism

### PR DESCRIPTION
**Pull request by Ragansu**
This is a new mechanism to launch  100 jobs as 1 job so, 
with this we should be ideally a around 100X faster.

RC

### CODE REVIEW

**Comment by Ragansu** [comment]
I have ran a test with this.  I will let you know if it worked, Then I will mark this PR for review

File: scripts/run_generator.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+##!/bin/bash
+
+export OutputDir=/global/cfs/cdirs/m4287/hep/Delphes_PYTHIA8_output
+export WorkDir=/global/cfs/cdirs/m4287/hep/genHEPdata
+
+shifter --image=rootproject/root:6.28.04-ubuntu22.04 /bin/bash $WorkDir/scripts/generate_events.sh $1 $2
```
Comment (sparticlesteve) [inline scripts/run_generator.sh]: you shouldn't need the `--image` option here if it is set in the job's `#SBATCH` declarations. Anyway, the image version should probably match.

**Comment by Ragansu** [comment]
right now I have set it to 4 nodes = 256 physical core and 2TB RAM,
with 512 in parallel I hope there wont be RAM issue.

**Comment by Ragansu** [comment]
After Taking note of how much data is needed and how jobs are allocated I decided to switch to 1 node with -n set to 64 and -c set to 4. 

With this we need a around 350 for generation of all the data. which can be achieved in a day. 

So ideally once we merge this PR in a days time new dataset can be made available.
For now I have confirmed that the DelphesPythia part is working fine. I have just launched the full chain (DelphesPythia8 + preprocess + deletion of root files) as test. will check it before merging this to main

Thank You @sparticlesteve and @wbhimji  for your help.

RC

**Pull request merged**